content/community.json

```diff
@@ -1,9 +1,18 @@
 {
-  "hero": {
-    "pictureDesktop": "/img/community/banner.png",
-    "pictureMobile": "/img/community/banner-mobile.png",
-    "url": "https://www.mlprague.com/#schedule-saturday"
-  },
+  "hero": [
+    {
+      "pictureDesktop": "/img/community/banner.png",
```

Comment:
we'll need to reconcile this as well ... I think @andronovhopf removed them on my request :( so, might want to get them back or remove this entry after testing

Reply:
Thanks for pointing it out! I think the reconciliation won't be too bad.
Having the old expired image was just for testing, I intended to remove it before merge anyway!